dags/bgbb.py

```diff
@@ -57,3 +64,74 @@
 )
 
 clients_daily_v6_dummy >> bgbb_fit
+
+
+subdag_args = default_args.copy()
+subdag_args["retries"] = 0
+task_id = "bgbb_fit_dataproc"
+params = get_dataproc_parameters("google_cloud_airflow_dataproc")
+
+bgbb_fit_dataproc = SubDagOperator(
+    task_id=task_id,
+    dag=dag,
+    subdag=moz_dataproc_pyspark_runner(
+        parent_dag_name=dag.dag_id,
+        dag_name=task_id,
+        job_name="bgbb_fit_dataproc",
+        cluster_name="bgbb-fit-{{ ds_nodash }}",
+        idle_delete_ttl="600",
+        num_workers=3,
+        worker_machine_type="n1-standard-8",
+        init_actions_uris=[
+            "gs://dataproc-initialization-actions/python/pip-install.sh"
+        ],
+        additional_properties={
+            "spark:spark.jars": "gs://spark-lib/bigquery/spark-bigquery-latest.jar"
+        },
+        additional_metadata={
+            "PIP_PACKAGES": "git+https://github.com/wcbeard/bgbb_airflow.git@bigquery"
```

Comment:
This needs to be changed once https://github.com/wcbeard/bgbb_airflow/pull/13 lands.